packages/host/src/lib/host.ts

```diff
@@ -918,44 +896,44 @@ export class Host implements IComponent {
      */
     // eslint-disable-next-line complexity
     async handleStartSequence(req: ParsedMessage): Promise<OpResponse<STHRestAPI.StartSequenceResponse>> {
-        try {
-            if (await this.loadCheck.overloaded()) {
-                return {
-                    opStatus: ReasonPhrases.INSUFFICIENT_SPACE_ON_RESOURCE,
-                };
-            }
+        if (await this.loadCheck.overloaded()) {
+            return {
+                opStatus: ReasonPhrases.INSUFFICIENT_SPACE_ON_RESOURCE,
+            };
+        }
 
-            const sequenceId = req.params?.id as string;
-            const payload = req.body || ({} as STHRestAPI.StartSequencePayload);
+        const sequenceId = req.params?.id as string;
+        const payload = req.body || ({} as STHRestAPI.StartSequencePayload);
 
-            if (payload.instanceId) {
-                if (!isStartSequenceEndpointPayloadDTO(payload)) {
-                    return { opStatus: ReasonPhrases.UNPROCESSABLE_ENTITY, error: "Invalid Instance id" };
-                }
+        if (payload.instanceId) {
+            if (!IDProvider.isValid(payload.instanceId)) {
```

Comment:
will fail tests. actually we allow any 36chars long instance id and tests use not uuid 36 long strings